.travis.yml

```diff
@@ -125,6 +130,8 @@ script:
   - bazel fetch //tensorboard/...
   - bazel build //tensorboard/...
   - bazel test //tensorboard/...
+  # Run manual S3 test
+  - bazel test //tensorboard/compat/tensorflow_stub:gfile_s3_test
```

Comment:
This test was added in the previous commit; shouldn’t any relevant CI
changes be in that commit as well? (Likewise for `boto3`/`moto`/config.)

Reply:
My idea was to add that test to be runnable locally in the other PR, but only activate it in Travis (with moto, boto3, etc) here. That being said, I'm not 100% sure on the split @nfelt was thinking, so happy to move it back. :)